Code/source/CallParser.cpp

```diff
@@ -0,0 +1,35 @@
+#include "CallParser.h"
+
+std::regex validCallStatement("\\s*call\\s+(\\w+)\\s*");
```

Comment:
I've checked online and \\w include underscore characters as well. Can you check if 
call a_b will pass this regex

Reply:
yes, all of them will pass. 123abc will pass as well.

I have wrote an additional check to check if it is a valid Procedure name.
The regex is just used to split the call and the procedure name.